### ISSUE TITLE
Add role=presentation to canonical resource stat block

### ISSUE THREAD

**Issue by ElliotSilver** [HL7/ig-template-base#258]
The stat block at the top of each canonical resource page uses a table despite not being tabular (i.e, row and column) data. To indicate to a screen reader that this use is only for layout, the table should be give the `role='presentation'` attribute.
